src/module.c

```diff
@@ -10863,6 +10865,16 @@ unsigned long long RM_CommandFilterGetClientId(RedisModuleCommandFilterCtx *fctx
     return fctx->c->id;
 }
 
+/* Get the indication of the command succeeded or failed */
+int RM_CommandFilterCmdIsSucceeded(RedisModuleCommandFilterCtx *fctx) {
+    return fctx->is_succeeded;
+}
+
+/* Get the indication of the data had been changeed */
+int RM_CommandFilterDataIsDirty(RedisModuleCommandFilterCtx *fctx) {
+    return fctx->is_dirty;
+}
+
```

Comment:
please improve the documentation a bit.
mention it should be called from within the post command callback (and the API it's registered with).